core/src/main/java/org/stellar/anchor/config/SecretConfig.java

```diff
@@ -1,20 +1,26 @@
 package org.stellar.anchor.config;
 
+import javax.crypto.SecretKey;
+
 public interface SecretConfig {
-  String getSep6MoreInfoUrlJwtSecret();
+  SecretKey getSep6MoreInfoUrlJwtSecret();
```

Comment:
This is a config interfaces and should be mapped to the configuration. Please use a class or helper function to avoid confusion.